README.md

```diff
@@ -1 +1,148 @@
 # cairo-coverage
+
+## Example
+
+Let's say you have a following program
+
+```rust
+pub enum Operation {
+    Add,
+    Multiply,
+}
+
+
+fn add(a: i32, b: i32) -> i32 {
+    a + b
+}
+
+fn multiply(a: i32, b: i32) -> i32 {
+    a * b
+}
+
+pub fn calculator(a: i32, b: i32, operation: Operation) -> i32 {
+    match operation {
+        Operation::Add => add(a, b),
+        Operation::Multiply => multiply(a, b),
+    }
+}
+
+```
+
+and you cover it with tests
+
+```rust
+#[test]
+fn calculator_add() {
+    assert_eq!(calculator(2, 3, Operation::Add), 5);
+    assert_eq!(calculator(-1, 1, Operation::Add), 0);
+}
+```
+
+When running with `cairo-coverage` you will get a coverage report in `.lcov` format:
+
+```lcov
+TN:
+SF:/path/to/your/project/src/lib.cairo
+FN:7,8,add
+FN:11,12,multiply
+FN:15,20,calculator
+FNDA:2,add
+FNDA:0,multiply
+FNDA:2,calculator
+FNF:3
+FNH:2
+DA:7,2
+DA:8,2
+DA:11,0
+DA:12,0
+DA:15,2
+DA:16,2
+DA:17,2
+DA:18,0
+DA:19,2
+LF:9
+LH:6
+end_of_record
+```
+
+Let's break it down
+
+### Explanation
+
+1. **General Information**
+    - **TN**: Test Name (optional, left empty)
+    - **SF**: Source File path `/path/to/your/project/src/lib.rs`
+
+2. **Function Summary**
```

Comment:
nit: Why is `Function Summary` labeled as 2. instead of 4. to align with the order in the example above?

Reply:
Fixed